internal/engine/wazevo/backend/isa/arm64/lower_instr.go

```diff
@@ -682,8 +688,8 @@ func (m *machine) LowerInstr(instr *ssa.Instruction) {
 		rn := m.getOperand_NR(m.compiler.ValueDefinition(x), extModeNone)
 		tmpReg := m.compiler.AllocateVReg(ssa.TypeI32)
 
-		// Our encoding for vecLoad1R does not support all the addressing modes yet,
-		// we use the no-offset addressing mode and add the offset to a temp register.
```

Comment:
it is not a matter of encoding, but we won't need post index load in the first place FYI @evacchi